### DISCUSSION TITLE
Starred Repository Tab Bug

### DISCUSSION THREAD

**Discussion by hustlewithvikram** [community/community#126427] [Mobile] [answered]
Steps to reproduce bug:

1. sign in to github mobile.
2. change your github username through a browser.
3. open the github mobile and tap on starred Repository Button
4. That's it you get a "Something went wrong" message.

![Screenshot_20240529-192613.jpg](https://github.com/community/community/assets/80647839/aa235394-8c4e-4696-9d93-f87880002e59)

~ as you can see the username is vikramv20 in the screenshot.

How i fixed it:

- Just Logout and Login again.

**Comment by taki-on** [comment, marked as answer]
@vikramisdev Thank you for reporting the bug. I will pass it to the Android team so that they can have a look.
